pkg/api/frontend_logging_test.go

```diff
@@ -45,8 +46,12 @@ func logSentryEventScenario(t *testing.T, desc string, event frontendlogging.Fro
 
 		sc := setupScenarioContext(t, "/log")
 
+		CDNRootURL, e := url.Parse("https://storage.googleapis.com/grafana-static-assets")
```

Comment:
Local variables shouldn't be capitalized.
```suggestion
		cdnRootURL, e := url.Parse("https://storage.googleapis.com/grafana-static-assets")
```

Reply:
fixed!